jdaviz/components/plugin_dataset_select.vue

```diff
@@ -15,15 +15,15 @@
     <template slot="selection" slot-scope="data">
       <div class="single-line">
         <span>
-          <v-icon style='margin-right: 2px'>{{ data.item.icon }}</v-icon>
+          <v-icon v-if="data.item.icon" style='margin-right: 2px'>{{ data.item.icon }}</v-icon>
```

Comment:
this just makes the left of the entries with and without icons line up better.  I also applied this across all other instances, even though not all dropdowns use "manual entries".